### PR TITLE
Modifying the Java build file to handle newly added MintLogger.java

### PR DESCRIPTION
**Pull request by kannappanr**
Modifying the Java build file to handle newly added MintLogger.java file, related to #597 update Minio-java to output new mint log format

### CODE REVIEW

**Comment by NitishT** [comment]
Should be merged after minio-java release 3.0.7

File: build/java/minio-java.sh [hunk 1]
```diff
@@ -27,12 +27,13 @@ install() {
     curl https://raw.githubusercontent.com/minio/minio-java/"${MINIO_JAVA_SDK_VERSION}"/functional/ContentInputStream.java > "${MINIO_JAVA_SDK_PATH}"/ContentInputStream.java
     curl https://raw.githubusercontent.com/minio/minio-java/"${MINIO_JAVA_SDK_VERSION}"/functional/PutObjectRunnable.java > "${MINIO_JAVA_SDK_PATH}"/PutObjectRunnable.java
     curl https://raw.githubusercontent.com/minio/minio-java/"${MINIO_JAVA_SDK_VERSION}"/functional/FunctionalTest.java > "${MINIO_JAVA_SDK_PATH}"/FunctionalTest.java
+    curl https://raw.githubusercontent.com/minio/minio-java/"${MINIO_JAVA_SDK_VERSION}"/functional/MintLogger.java > "${MINIO_JAVA_SDK_PATH}"/MintLogger.java
```
Comment (balamurugana) [inline build/java/minio-java.sh]: Multiple curl commands approach is not scalable.   We could use `svn` to download all files in functional test directory like

```bash
svn export https://github.com/minio/minio-java/trunk/functional
```

**Comment by balamurugana** [comment]
@kannappanr if this PR is not applicable, then close it

**Comment by kannappanr** [comment]
Closing this PR as https://github.com/minio/mint/pull/128 will handle the changes intended by this PR.

**Pull request closed without merging**